build.gradle

```diff
@@ -177,8 +177,9 @@ dependencies {
     testCompile group: 'com.github.tomakehurst', name: 'wiremock-standalone', version: '2.21.0'
     testCompile group: 'org.springframework.cloud', name: 'spring-cloud-contract-wiremock', version: '2.1.0.RELEASE'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.3'
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.2-Categories'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.7-CCD-3694_CCD-3836'
+    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.7-CCD-3694_CCD-3836'
```

Comment:
Should be updated with proper release number once hmcts/befta-fw#192 is released.